packages/edit-site/src/components/post-edit-header/index.js

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Nit: I wonder if this deserves its own folder or if it could be just a `header.js` file within the `post-edit`. 

Personally and given the CSS is smallish, I could have kept it in the same folder.